src/Payum/LaravelPackage/PayumServiceProvider.php

```diff
@@ -21,32 +21,11 @@ class PayumServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $this->package('payum/payum-laravel-package');
-        \View::addNamespace('payum/payum', __DIR__.'/../../views');
-
-        $this->app->error(function(ReplyInterface $reply)
```

Comment:
I'd prefer this way, cuz you can easily customize response just by registering another listener